### ISSUE TITLE
E-mail turned off warning when changing user password

### ISSUE THREAD

**Issue by selfhoster1312** [YunoHost/issues#2226]
### Describe the bug

Creating an account without email permission works great. When the password is changed from the admin UI, there is a warning: `E-mail turned off for user x`

![image](https://github.com/YunoHost/issues/assets/121760708/96c89fe2-72a1-4497-aa97-8d6a11a06589)

**Comment by selfhoster1312** [comment]
Usecase: i don't want to give email permission to every user because we use the server mostly for local sharing of documents, and i don't want the server to end up on spam lists because a user has a bad password.

I think it's important for admins to be able to receive emails, but for other users not really? At least not if we use Nextcloud/XMPP and not email for that.

I opened a pull request here: https://github.com/YunoHost/yunohost/pull/1739

I'm not sure it's the best solution, especially because the warning message should be updated if that's the solution we chose.

What do other people think?
